bokehjs/src/lib/models/widgets/spinner.ts

```diff
@@ -104,9 +105,9 @@ export class SpinnerView extends NumericInputView {
   }
 
   get precision(): number {
-    const {low, high, step} = this.model
+    const {low, high, step, initial_value_precision} = this.model
```

Comment:
Do we actually need to save and and only use the precision from the initial value? Would it not simply work to always look at the current value? (if the initial value is a float then adding an int to it should not change its precision, for not-extremely large values at least, and if we are in that regime we won't be able to honor the step anyway)

Reply:
If I understand you correctly, you mean to apply the precision on this.model.value directly? if so then I actually did do that in the begging (see first commit) but that meant floating errors in some cases, for instance when decrementing, the value would end up showing as 1.299999999998 or so. Please correct me if I am wrong

Reply:
Ex: Spinner(value=1.3, low=0, high=10, mode="float", step=1)

increment once: value = 2.3
decrement once: value = 1.2999999999999998

Reply:
Properties are meant for configurability or for computed values of significance to the user, so if that's not the case then this shouldn't be a public property, and possibly a property altogether. If we really need to store this, then a view private member should be sufficient.

Reply:
If it isn't stored do you suggest doing this another way? As I mentioned , taking the value's precision right away resulted in large precisions, or am I doing something wrong?

Thank you for your feedback

Reply:
I would definitely be more comfortable with computing an initial max_precision if that fed into a formatter, rather than actually futzing with values.